docs/helm/oathkeeper.md

```diff
@@ -91,3 +91,15 @@ $ helm install \
     --set 'maester.enabled=false' \
     ory/oathkeeper
 ```
+
+#### Operation modes
+The Maester controller is now available in two modes: 
+- global controller
+- dedicated sidecar
+
+**Controller mode**
+In this mode, the controller is a dedicated deployment, and scales separately of the oathkeeper application. All communication with oathkeeper is based on a configMap object, which stores the translated oathkeeper configuration. 
```

Comment:
```suggestion
In this mode, the controller is a dedicated deployment and scales independently from the Oathkeeper application. All communication with Oathkeeper is based on a configMap object, which stores the translated Oathkeeper configuration. 
```

`which stores the translated` -> this is not exactly clear to me. I'm guessing it's translated from "oathkeeperish" to "Kuberneres"? Can you rephrase it and make it clearer pls?